packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -292,9 +261,9 @@ export function FileSnapshotWriterClassFactory<TBase extends ReaderConstructor>(
             }
 
             for (const blobName of Object.keys(snapshotTree.blobs)) {
-                const contents = await this.read(snapshotTree.blobs[blobName]);
+                const contents = blobToString(await this.readBlob(snapshotTree.blobs[blobName]));
```

Comment:
You did equivalent change.
But this code was not entirely correct, as it assumes all buffers are strings, not binary.
If you have other places where you need to convert binary blob to IBlob, it might be great to create helper function to do such conversion, and open an issue to follow up.
The follow up is 
1. fix it in there (by encoding into base64)
2. I future, once we switch IBlob to use buffers as well, change this helper function too.